internal/controllers/rest_controller.go

```diff
@@ -186,3 +193,17 @@ func (rc RESTController) forbiddenResponse(w http.ResponseWriter, err error) {
 	}
 	json.NewEncoder(w).Encode(jsonBody)
 }
+
+// allowlistMiddleware is a middleware that allows all requests that match the allowlist.
+func (rc RESTController) allowlistMiddleware() gin.HandlerFunc {
+	return func(c *gin.Context) {
+		for _, allow := range rc.configuration.AllowlistURL {
+			pathPrefix := fmt.Sprintf("/%s", strings.TrimPrefix(allow, "/"))
```

Comment:
Did you remove the `/` with `TrimPrefix` and add `/` again with `Sprintf`?

Reply:
Yes, I did to ensure a typo passed by the user, e.g: `dex/`

Reply:
nonsense... I'll remove it.

Reply:
This line is not informed by the user, it is a configuration.